tests/client-dbus/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
I think you can just get rid of the LICENSE file. It made more sense when this was actually a library.